website/pages/docs/connect/terminating_gateway.mdx

```diff
@@ -0,0 +1,99 @@
+---
+layout: docs
+page_title: Connect - Terminating Gateways
+sidebar_title: Terminating Gateways
+description: >-
+  A Terminating Gateway enables traffic from services in the Consul
+  service mesh to services outside the mesh. This section details
+  how to configure and run a terminating gateway.
+---
+
+# Terminating Gateways
+
+-> **1.8.0+:**  This feature is available in Consul versions 1.8.0 and newer.
+
+Terminating gateways enable connections from services in the Consul service mesh to
+services outside the mesh. These gateways effectively act as Connect proxies that can
+represent more than one service. They terminate Connect mTLS connections, enforce intentions,
+and forward requests to the appropriate destination.
+
+For data that must be encrypted in transit to the final destination, terminating gateways
+support TLS origination. By specifying a [CA file](/docs/agent/config-entries/terminating-gateway#cafile),
+[client certificate](/docs/agent/config-entries/terminating-gateway#certfile), and
+[private key](/docs/agent/config-entries/terminating-gateway#keyfile),
+connections from the gateway to a linked service can be encrypted using one-way or mutual TLS.
+If none of these are provided, Consul will only encrypt connections to the gateway and not
+from the gateway to the destination service. 
+
+-> **Note:** If certificates and keys are configured the gateway will upgrade HTTP connections to TLS. 
+Client applications can issue plain HTTP requests even when connecting to servers that require HTTPS.
+
+## Prerequisites
+
+Each terminating gateway needs two things:
```

Comment:
```suggestion
Each terminating gateway needs:
```

Reply:
Do we also want to say it needs a supported proxy technology?

Reply:
That's mentioned a little farther down:
https://github.com/hashicorp/consul/pull/7710/files#diff-bcc0b2977dadf67996a8a8ce825314a8R45